### PR TITLE
Fix NFT skeletons on non-NFT projects

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

**Test:**
- non-NFT project: `/v2/p/1`
- NFT project: `/v2/p/4633`

Also addresses your comment about flipping the ternary in from #1815 

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/pages/v2/p/components/V2Dashboard.tsx [hunk 1]
```diff
@@ -176,7 +177,8 @@ export default function V2Dashboard({
   //    - loading will be true briefly
   //    - will resolve false when `useNftRewardTiersOf` fails
   const nftsLoading = Boolean(
-    fundingCycleMetadata?.dataSource &&
+    fundingCycleMetadata &&
+      fundingCycleMetadata?.dataSource !== constants.AddressZero &&
```
Comment (tomquirk) [inline src/pages/v2/p/components/V2Dashboard.tsx:181]:
```suggestion
      fundingCycleMetadata?.dataSource !== constants.AddressZero &&
```

Comment (tomquirk) [inline src/pages/v2/p/components/V2Dashboard.tsx:181]:
```suggestion
    fundingCycleMetadata?.dataSource &&
      fundingCycleMetadata.dataSource !== constants.AddressZero &&
```

Comment (johnnyd-eth) [reply src/pages/v2/p/components/V2Dashboard.tsx]: Addressed

File: src/pages/v2/p/components/V2Dashboard.tsx [hunk 2]
```diff
@@ -177,6 +178,7 @@ export default function V2Dashboard({
   //    - will resolve false when `useNftRewardTiersOf` fails
   const nftsLoading = Boolean(
     fundingCycleMetadata?.dataSource &&
+      fundingCycleMetadata?.dataSource !== constants.AddressZero &&
```
Comment (tomquirk) [inline src/pages/v2/p/components/V2Dashboard.tsx:181]:
```suggestion
      fundingCycleMetadata.dataSource !== constants.AddressZero &&
```

**Review by tomquirk — APPROVED** [review]

**Pull request merged**